Sources/Testing/Events/Clock.swift

```diff
@@ -338,3 +338,41 @@ extension Test.Clock.Instant {
 #endif
   }
 }
+
+@available(_clockAPI, *)
+extension Test.Clock.Instant: Codable {
+  private enum CodingKeys: CodingKey {
+    case suspending
+    case wall
+  }
+
+  public func encode(to encoder: any Encoder) throws {
+    var container = encoder.container(keyedBy: CodingKeys.self)
+#if SWT_TARGET_OS_APPLE
+    let suspending = SuspendingClock.Instant(self)
```

Comment:
A quick comment here explaining what we're doing here (making sure there's a consistent cross-platform encoding) would be good.